.travis.yml

```diff
@@ -52,9 +52,9 @@ script:
   - flake8 --config=.flake8.cython .
   - autopep8 -r . --global-config .pep8 | tee check_autopep8
   - test ! -s check_autopep8
-  - (for NP in 1 2 3; do PYTHONWARNINGS='ignore::FutureWarning,module::DeprecationWarning' mpiexec -n ${NP} nosetests -v -a '!nccl,!gpu,!slow' || exit $?; done)
-  # - cd tests
-  # - PYTHONWARNINGS='ignore::FutureWarning,module::DeprecationWarning' nosetests -a '!gpu,!slow' --with-doctest chainer_tests
+  - if mpiexec --version | grep -q OpenRTE; then NOBIND="--bind-to none"; else NOBIND= ; fi
+  - (for NP in 1 2; do for T in $(find tests -name "test_*.py") ; do mpiexec -n ${NP} nosetests -s -v -a '!nccl,!gpu,!slow' $T || exit $?; done; done)
```

Comment:
Why are the tests run separately?
I think it is not needed to change.

Reply:
This was also "safety guard" for future potential errors... but I removed it for the same reason.